### PR TITLE
feat: update the lab 5 instructions to remove redux

### PR DESCRIPTION
**Pull request by JakePartusch**

### CODE REVIEW

**Comment by DSchau** [comment]
Deploy preview for *react-redux-workshop* ready!

Built with commit fc078b3bf7cc21302095062e3a1fb32d9ce752a8

https://deploy-preview-79--react-redux-workshop.netlify.com

File: labs/content/labs/05-forms.md [hunk 1]
```diff
@@ -180,48 +180,26 @@ initialValues={ {
...
 ```jsx
-const mapDispatchToProps = {
-  onCreate: EmployeeActionCreators.createEmployee,
-  onUpdate: EmployeeActionCreators.updateEmployee,
-  getEmployee: EmployeeActionCreators.getEmployee
-};
+  async componentDidMount() {
+    const { match } = this.props;
+    const { _id } = match.params;
+    const { data: employee } = await Axios.get(url(_id));
```
Comment (DSchau) [inline labs/content/labs/05-forms.md]: Where is this url function coming from?

🤔

Comment (JakePartusch) [reply labs/content/labs/05-forms.md]: First of all, whatcha doin in here? 😂 Second, the function is already in the file that they are editing.. Might be a bit confusing though?

Comment (DSchau) [reply labs/content/labs/05-forms.md]: I've found that ~75% of people just copy and paste--so if it makes sense in context that's probably OK!

And 🤷‍♂️

File: labs/content/labs/05-forms.md [hunk 2]
```diff
@@ -244,47 +221,56 @@ render() {
...
-          }}
-        />
-      </MemoryRouter>
-    </Provider>
-  );
-});
-
-it('should instantiate the Employee Detail Component', () => {
-  expect(wrapper.find(EmployeeDetail)).toIncludeText('Employee Detail');
+jest.mock('axios', () => ({
```
Comment (mike-plummer) [inline labs/content/labs/05-forms.md]: Other unit tests are using moxios, but I prefer using pure Jest mocks. Maybe something to standardize in the future?

**Pull request merged**